### PR TITLE
Dls 8828 - Creating Balance Summary For Correct Return Check Changes

### PR DESCRIPTION
**Pull request by JakeReid2020**

### CODE REVIEW

File: app/views/helpers/AmountToPaySummary.scala [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    val originalReturnTotal: BigDecimal = amounts.originalReturnTotal
+    val newReturnTotal: BigDecimal = amounts.newReturnTotal
+    val accountBalance: BigDecimal = amounts.accountBalance
+    val adjustedAmount: BigDecimal = amounts.adjustedAmount
+
+    SummaryListViewModel(rows = Seq(
+      SummaryListRowViewModel(
+        key = "correctReturn.originalReturnTotal",
+        value = ValueViewModel(HtmlContent(CurrencyFormatter.formatAmountOfMoneyWithPoundSign(originalReturnTotal).replace("-", "&minus;")))
+          .withCssClass("original-return-total sdil-right-align--desktop")
```
Comment (drewmiley) [inline app/views/helpers/AmountToPaySummary.scala:39]: Should `original-return-total` be a class? I think it should be an id?
Similar for the other three below

File: app/orchestrators/CorrectReturnOrchestrator.scala [hunk 2]
```diff
@@ -20,8 +20,11 @@ import cats.data.EitherT
 import com.google.inject.{Inject, Singleton}
 import connectors.SoftDrinksIndustryLevyConnector
 import errors.{FailedToAddDataToUserAnswers, NoSdilReturnForPeriod, NoVariableReturns}
+import models.core.ErrorModel
```
Comment (drewmiley) [inline app/orchestrators/CorrectReturnOrchestrator.scala:23]: Remove unneeded imports

File: app/controllers/correctReturn/CorrectReturnCheckChangesCYAController.scala [hunk 3]
```diff
@@ -35,24 +36,43 @@ import scala.concurrent.{ExecutionContext, Future}
...
+
+        def sections(accountBalance: BigDecimal): Seq[(String, SummaryList)] = {
+          CorrectReturnCheckChangesSummary.changeSpecificSummaryListAndHeadings(
+            request.userAnswers,
+            request.subscription,
+            changedPages,
+            amounts = Amounts(
+              originalReturnTotal = originalSdilReturn.total,
+              newReturnTotal = SdilReturn(request.userAnswers).total,
+              accountBalance = accountBalance * -1,
```
Comment (drewmiley) [inline app/controllers/correctReturn/CorrectReturnCheckChangesCYAController.scala:61]: Is there a better name for one of the `accountBalance`?  This looks unusual

File: app/connectors/SoftDrinksIndustryLevyConnector.scala [hunk 4]
```diff
@@ -83,38 +83,30 @@ class SoftDrinksIndustryLevyConnector @Inject()(
   def balance(
                sdilRef: String,
                withAssessment: Boolean
-             )(implicit hc: HeaderCarrier): VariationResult[BigDecimal] = EitherT {
```
Comment (drewmiley) [inline app/connectors/SoftDrinksIndustryLevyConnector.scala:86]: The return types of these have been changed. Are they used by anywhere else in the service?

Comment (JakeReid2020) [reply app/connectors/SoftDrinksIndustryLevyConnector.scala:86]: No these are only used this pages so far

**Review by drewmiley — APPROVED** [review]

**Pull request merged**